rdmo/views/templatetags/view_tags.py

```diff
@@ -111,6 +111,33 @@ def get_set(context, attribute, set_prefix='', project=None):
     return get_sets(context, attribute, set_prefix=set_prefix, project=project)
 
 
+@register.simple_tag
+def render_custom_inline_list(values=None, sep=',', sep_last=', and'):
+    values = values or []
+
+    if not values:
+        return ''
+
+    if len(values) == 1:
+        return values[0]['value_and_unit']
+
+    if len(values) == 2:
+        if sep_last.startswith(', '):  # for english sep_last
+            sep_last = sep_last.split(', ')[-1]
+        return f"{values[0]['value_and_unit']} {sep_last} {values[1]['value_and_unit']}"
+
+    _text = ''
+    for n,value in enumerate(values):
+        if n == len(values) - 1 and _text:
+            _text += f"{sep_last} "
+        _text += value['value_and_unit']
+        if n < len(values) - 2:
+            _text += f"{sep} "
+        else:
+            _text += ' '
+    return _text
+
+
```

Comment:
Ignoranter Versuch, nicht getestet: können wir vielleicht in der `if`-Bedingung nur den Separator ersetzen?

```suggestion
    if len(values) == 2:
        separator_last = separator_last.replace(', ',' ')  # for english sep_last

    text = ''
    for n,value in enumerate(values):
        if n == len(values) - 1 and text:
            text += f"{separator_last} "
        _text += value['value_and_unit']
        if n < len(values) - 2:
            text += f"{separator} "
        else:
            text += ' '
return text


```